prov/sm2/src/sm2_progress.c

```diff
@@ -138,6 +186,11 @@ static int sm2_start_common(struct sm2_ep *ep,
 			xfer_entry, (struct ofi_mr **) rx_entry->desc,
 			rx_entry->iov, rx_entry->count, &total_len, ep, 0);
 		break;
+	case sm2_proto_ipc:
+		err = sm2_progress_ipc(
+			xfer_entry, (struct ofi_mr **) rx_entry->desc,
```

Comment:
Format looks funky

Reply:
We are all at the mercy of clang-format gods

Reply:
The formatting actually got fixed in a later commit